yellowbrick/features/base.py

```diff
@@ -75,11 +65,68 @@ def fit_transform_poof(self, X, y=None, **kwargs):
         return Xp
 
 
+class MultiFeatureVisualizer(FeatureVisualizer):
+    """
+    MultiFeatureVisualiers are a subclass of FeatureVisualizer that visualize
+    several features at once. This class provides base functionality for
+    getting the names of features for use in plot annotation.
+
+    Parameters
+    ----------
+
+    ax: matplotlib Axes, default: None
+        The axis to plot the figure on. If None is passed in the current axes
+        will be used (or generated if required).
+
+    features: list, default: None
+        a list of feature names to use
+        If a DataFrame is passed to fit and features is None, feature
+        names are selected as the columns of the DataFrame.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    """
+
+    def __init__(self, ax=None, features=None, **kwargs):
```

Comment:
I would prefer if features was renamed to labels to be more consistent with Scikit-Learn. This probably involves cleaning up some other code. Also, labels should also accept a LabelEncoder. 

You don't have to do it in this PR, but would you mind opening up an issue with this and mention the locations of where things might have to change?